howto/vscode-debugging-daprd/README.md

```diff
@@ -0,0 +1,150 @@
+# Configuring Visual Studio Code for debugging with daprd
+
+While developing locally in VSCode, you might find it useful to be able to attach the debugger to your code that is utilizing Dapr. To do this, you can run the daprd process separately and then launch your code and attach the debugger. While this is a perfectly acceptable solution, it does require a few extra steps and some instruction to developers who might want to clone your repo and hit the "play" button to begin debugging.
+
+Using the ***tasks.json*** and ***launch.json*** files in Visual Studio Code, we can simplify the process and request that VSCode kick off the daprd process prior to launching the debugger.
```

Comment:
Add links to https://code.visualstudio.com/Docs/editor/debugging for the launch.json and https://code.visualstudio.com/Docs/editor/tasks for the tasks.json if people need to find out more.

Also prefer using the word "you" over "we". For example "you can simplify the process"